config/routes.rb

```diff
@@ -12,6 +12,12 @@
 
   get 'sentiment' => 'sentiment#check'
 
+  get 'favorite' => 'favorite#favorite'
+
+  post 'favorite/create/:company_id' => 'favorite#create_favorite', as: :create_favorite
```

Comment:
I use these when calling the path. Its better practice to have rails generate a definition for the path. That way, we only change it in routes.rb and not all the files where we use the path. For example, I use it in the following
          <%= button_to remove_favorite_path(res.id), method: :patch, remote: true, class: "btn btn-default" do%>
This way if we do go back and change the path, we dont need to go and change all instances.